src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -656,16 +661,21 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 		onDidChangeCursorPosition();
 	}
 
-	private initAttachedContext(container: HTMLElement, isLayout = false) {
+	private async initAttachedContext(container: HTMLElement, isLayout = false) {
+		this.attachedContextDisposables.clear();
```

Comment:
This is redundant right?

Reply:
Yes you're right, it's cleared out in the .value set